### ISSUE TITLE
Can not install via gemfile in Rails 3.0?

### ISSUE THREAD

**Issue by peterk** [fidel/batch_translations#3]
When adding "batch_translations" to a default Rails 3.0 project and running bundle install you get the error:

```
Could not find gem 'batch_translation (>= 0)' in any of the gem sources listed in your Gemfile.
```

Adding the github repo does not work either. It would be awesome if you could make it possible to install batch_translations via the standard mechanism.

**Comment by lucapette** [comment]
Yep I'd prefer to use it as a gem too. @fidel have you planned to make a gem from this nice job?

**Comment by fidel** [comment]
Sorry guys, but I don't have time to maintain this plugin at the moment.

**Comment by lucapette** [comment]
I understand perfectly. I can't promise now it but I could do a pull request with the necessary configuration. However, you should take some time after that pull request to configure the gemspec. If it sounds good I'll try to do it in the next week.

**Comment by fidel** [comment]
Ok, we can try to do that this way.
